src/constants.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 export class Constants {
-    public static DesignerVersion = "1.40128.1.415871.190130-1749";
+    public static DesignerVersion = "1.40211.1.1557826.190208-1341";
```

Comment:
I'll be updating this version number as we do more staging deployments.  1.40211.1.2 is already in progress.